src/components/activity/ManageActivities.jsx

```diff
@@ -66,8 +67,56 @@ const ManageActivities = ({ admin, history }) => {
                   {activity.title}
                 </summary>
                 <p className="AdminDiscription">{activity.description}</p>
+                <button
+                  type="button"
+                  onClick={
+                    () => {
+                      Swal.fire({
+                        title: 'Changer le Titre',
+                        input: 'text',
+                        inputValue: activity.description,
+                        inputAttributes: {
+                          autocapitalize: 'off',
+                        },
+                        showCancelButton: true,
+                        cancelButtonText: 'Annuler',
+                        preConfirm: (newDescription) => {
+                          Axios.put(`http://localhost:8000/activities/${activity.id}`, { description: newDescription })
```

Comment:
looks like this is changing the description, while the title says "Changer le titre"